<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidiInputSettingsDialog</class>
 <widget class="QDialog" name="MidiInputSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Note: This functionality depends on the 'miditcp' server program, included in the DViz source directory. Currently, the miditcp program only runs on linux. Make sure it's running if you're having problems with this dialog.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>MIDI TCP Server:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostEdit">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="portBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
       <property name="value">
        <number>3729</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hostApplyBtn">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="connectStatusLabel">
       <property name="text">
        <string>(Unknown)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>DViz Action</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Input Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MIDI Key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Current Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="changeBtn">
       <property name="toolTip">
        <string>To change a key for a row, first click on the row to change and click &quot;Change Key&quot;</string>
       </property>
       <property name="text">
        <string>Change Key for Selected Row</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MidiInputSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MidiInputSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
